Configuration/PyReleaseValidation/test/test_mc_setup/test_MC_setup.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+# This script mimics a MC chain with HLT run 
+# a in a different release from the current one
+
+echo '#### MC Setup'
+
+# this would make it run only in 14_0_X releases, usefull?
+
+# if [[ ! ("$CMSSW_VERSION" == *"14_0_"*) ]]; then
+#     echo "This is not a 14_0_X release. No need to test this. Exiting."
+#     exit 0
+# fi
```

Comment:
P.S. this implies that I would backport this to `14_0_X`

Reply:
I think as long as there is any chance of a future Run3 UL (or any other future production) where this "HLT step is run in 12_4_X/13_0_X/..., and it needs to read the input from a modern release" behavior is expected, these tests should be kept in `master`. Otherwise it would be sufficient to have the test only in release cycles where this behavior is needed (i.e. as far as I am concerned, if starting from release cycle N this behavior will not be needed, the test can be removed in N).

Reply:
I agree with @makortel that the test should target the master. Backport to 14_0 is special case in this PR.

Reply:
Agreed!